spec/chitter_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'chits'
+describe Chits do
+    describe '.all' do
+      it 'returns all chitters' do
```

Comment:
You switch between 'chitter', 'chitters', 'chits' and 'chit' a lot! This would get confusing for someone else reading it.